data-truth/zoltar-truth-data.py

```diff
@@ -97,9 +97,9 @@ def configure_JHU_data(df, target):
     # find week-ahead targets
     for i in range(20):
         weeks_ahead = i + 1  # add one to [0,3]
-        days_back = 5 + ((weeks_ahead - 1) * 7)  # timezero is on Mondays
+        days_back = (weeks_ahead * 7)  # timezero is on Mondays
 
-        df_calc = df_truth_values  # initialize df
+        df_calc = df_truth_long  # initialize df
```

Comment:
what was the original intent of using Saturday truth values? Just curious.

Reply:
It's a convention in epidemiological surveillance data to define each week as Sunday through Saturday. An "MMWR week".